docs/standards/news.rst

```diff
@@ -0,0 +1,106 @@
+.. this Changelog is based on the merged pull requests involving the ````airr-schema.yaml```` file since Jan 9 2018
+
+.. toctree::
+   :maxdepth: 1
+   :caption:  AIRR Standards Release Notes
+
+Release Notes
+==============
+
+
+Version 1.2.?: current
+--------------------------------------------------------------------------------
+
++ Expanded description for ``pcr_target``
+ .. #288 by bussec was merged on Dec 10, 2018
++ Fix capitalization of ``age_unit`` description
+ .. #290 by bcorrie was merged on Dec 10, 2018
++ Fix name: fields for ``age`` fields
+ .. #285 by bcorrie was merged on Nov 27, 2018
++ Reverted ``locus_species`` until we resolve how to handle ontologies
```

Comment:
This might be confusing to a reader, as it partially undoes the changes documented in line 22. Maybe it's better to just comment on the net result (i.e. introduction on `cell_species`) an link both PRs in the comments.